api/handlers.go

```diff
@@ -52,6 +70,29 @@ func CreateSessionHandlerFunc(sessions Sessions) http.HandlerFunc {
 
 		w.Header().Set("Content-Type", "application/json")
 		w.WriteHeader(http.StatusCreated)
-		w.Write(sessJson)
+		w.Write(sessJSON)
+	}
+}
+
+func GetSessionHandlerFunc(cache Cache, getVarsFunc GetVarsFunc) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		ctx := r.Context()
+		ID := getVarsFunc(r)["ID"]
+
+		log.Event(ctx, fmt.Sprintf("get session by ID: %s", ID), log.INFO)
```

Comment:
Probably a good idea to remove this as we should n**ever** log out the session ID in production. If someone were able obtain one they would be able access things they shouldn't. Best to delete it now before we forget.